yayson/yayson.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for yayson v2.0.1
+// Project: https://github.com/confetti/yayson
+// Definitions by: David Wood <https://github.com/Codesleuth>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'yayson' {
+  class Store {
+    sync(obj: Object): void;
```

Comment:
I think they're identical in this context. (The difference would be that `any` isn't typechecked in the function's body, but this has no body.) Don't know which one is preferred.